client/proto/story/evmstaking/v1/types/params.proto

```diff
@@ -19,4 +19,10 @@ message Params {
   string ubi_withdraw_address = 4 [
     (gogoproto.moretags) = "yaml:\"ubi_withdraw_address\""
   ];
+  uint32 refund_fee_bps = 5 [
+    (gogoproto.moretags) = "yaml:\"refund_fee_bps\""
+  ];
+  uint32 refund_period = 6 [
+    (gogoproto.moretags) = "yaml:\"refund_period\""
+  ];
```

Comment:
Why don't use second for the unit of `refund_period`? It would be consistent with the unit of unbonding period.

Reply:
changed in 04cae7d to use `time.Duration`, which is the unit used in `unbonding_time` in the staking module ([ref](https://github.com/piplabs/cosmos-sdk/blob/c7845d7a9d54010e33b31f0d4267dedc3e6a0f96/proto/cosmos/staking/v1beta1/staking.proto#L388-L390))